src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/BulkCopyAEErrorMessage.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted.Setup;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
+{
+    /// <summary>
+    /// Always Encrypted public API Manual tests.
+    /// TODO: These tests are marked as Windows only for now but should be run for all platforms once the Master Key is accessible to this app from Azure Key Vault.
+    /// </summary>
+    [PlatformSpecific(TestPlatforms.Windows)]
+    public class BulkCopyAEErrorMessage : IClassFixture<SQLSetupStrategyCertStoreProvider>, IDisposable
+    {
+        private SQLSetupStrategyCertStoreProvider fixture;
+        
+        private readonly string _tableName;
+        private readonly string _columnName;
+
+        public BulkCopyAEErrorMessage(SQLSetupStrategyCertStoreProvider fixture)
+        {
+            this.fixture = fixture;
+            _tableName = fixture.BulkCopyAEErrorMessageTestTable.Name;
+            _columnName = "c1";
+        }
+
+        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringSetupForAE))]
+        [ClassData(typeof(AEConnectionStringProvider))]
+        public void TextToIntErrorMessageTest(string connectionString)
+        {
+            string value = "stringValue";
+            DataTable dataTable = CreateTable(value);
+
+            Assert.True(StringToIntTest(connectionString, _tableName, dataTable, value, dataTable.Rows.Count), "Did not get any exceptions for DataTable!");
+            Assert.True(StringToIntTest(connectionString, _tableName, dataTable.Select(), value, dataTable.Rows.Count),"Did not get any exceptions for DataRow[]!");
+        }
+
+        private DataTable CreateTable(string value)
```

Comment:
Nit: Could your rename this to `CreateDataTable()` so it's clear you're not actually creating database table here.